### ISSUE TITLE
What is the "causal_label" in the source code?

### ISSUE THREAD

**Issue by ghost** [shengyuzhang/DeVLBert#4]
`causal_prediction_v2t_loss, causal_prediction_t2v_loss= self.cls(
                sequence_output_t, sequence_output_v, pooled_output_t, pooled_output_v,
                causal_sequence_output_v, causal_sequence_output_v2t, causal_label_v, image_target,
                causal_sequence_output_t, causal_sequence_output_t2v, causal_label_t
            )`
Thanks for your great code.
I noticed that you calculate the `causal prediction loss` , however I couldnot find the causal label where it defined. Would you please help me?

**Comment by jiangtann** [comment]
At [line 24](https://github.com/shengyuzhang/DeVLBert/blob/5922879202de89a96d8eab33691c1398a7b7a3a3/train_concap.py#L24), we import `ConceptCapLoaderTrain` and `ConceptCapLoaderVal`, where we define at [DeVLBert/devlbert/datasets/concept_cap_dataset.py](https://github.com/shengyuzhang/DeVLBert/blob/5922879202de89a96d8eab33691c1398a7b7a3a3/devlbert/datasets/concept_cap_dataset.py#L190). You can find how we generate causal label there (search `causal_label_t` and `causal_label_v` at the page).
